add-ons/concourse/external-workers.yml

```diff
@@ -0,0 +1,82 @@
+jobs:
+- name: concourse_workers
+  plan:
+  - aggregate:
+    - get: stemcell
+    - get: bosh-deployment
+    - get: concourse-deployment
+  - put: cloud-config
+    params:
+      manifest: bosh-deployment/((bosh_cpi))/cloud-config.yml
+      releases: []
+  - put: deploy
+    params:
+      manifest: concourse-deployment/cluster/external-worker.yml
+      stemcells: [ stemcell/stemcell.tgz ]
+      vars:
+        deployment_name: concourse_workers
+        concourse_version: 3.8.0
+        concourse_sha1: 99e134676df72e18c719ccfbd7977bd9449e6fd4
+        garden_runc_version: 1.9.4
+        garden_runc_sha1: 9cccd7685ac075ad6956cba3ab5881e3435cd7e3
+        instances: 1
+        azs: [z1, z2, z3]
+        worker_vm_type: default
+        external_worker_network_name: default
+        worker_tags:
+        tsa_host: ((concourse_tsa_host))
+        tsa_host_key.public_key: ((concourse_tsa_host_key.public_key))
```

Comment:
why not `tsa_host_key: ((concourse_tsa_host_key))`? more consistent with concourse_worker_key